src/190B20.c

```diff
@@ -1074,7 +1074,13 @@ INCLUDE_ASM(s32, "190B20", func_80263300);
 
 INCLUDE_ASM(s32, "190B20", btl_are_all_enemies_defeated);
 
-INCLUDE_ASM(s32, "190B20", btl_check_enemies_defeated);
+s32 btl_check_enemies_defeated(void) {
+    if (btl_are_all_enemies_defeated() != 0) {
```

Comment:
Because this function you're calling returns a bool, you can remove the `!= 0` from the conditional here.

Reply:
I promise I'm a competent developer 😂